sdk/postgresql/Azure.ResourceManager.PostgreSql/src/PostgreSqlFlexibleServers/Generated/PostgreSqlFlexibleServerActiveDirectoryAdministratorData.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using Azure.Core;
+using Azure.ResourceManager.Models;
+using Azure.ResourceManager.PostgreSql.FlexibleServers.Models;
+
+namespace Azure.ResourceManager.PostgreSql.FlexibleServers
+{
+    /// <summary> A class representing the PostgreSqlFlexibleServerActiveDirectoryAdministrator data model. </summary>
+    public partial class PostgreSqlFlexibleServerActiveDirectoryAdministratorData : ResourceData
+    {
+        /// <summary> Initializes a new instance of PostgreSqlFlexibleServerActiveDirectoryAdministratorData. </summary>
+        public PostgreSqlFlexibleServerActiveDirectoryAdministratorData()
+        {
+        }
+
+        /// <summary> Initializes a new instance of PostgreSqlFlexibleServerActiveDirectoryAdministratorData. </summary>
+        /// <param name="id"> The id. </param>
+        /// <param name="name"> The name. </param>
+        /// <param name="resourceType"> The resourceType. </param>
+        /// <param name="systemData"> The systemData. </param>
+        /// <param name="principalType"> The principal type used to represent the type of Active Directory Administrator. </param>
+        /// <param name="principalName"> Active Directory administrator principal name. </param>
+        /// <param name="objectId"> The objectId of the Active Directory administrator. </param>
+        /// <param name="tenantId"> The tenantId of the Active Directory administrator. </param>
+        internal PostgreSqlFlexibleServerActiveDirectoryAdministratorData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, PostgreSqlFlexibleServerPrincipalType? principalType, string principalName, string objectId, Guid? tenantId) : base(id, name, resourceType, systemData)
+        {
+            PrincipalType = principalType;
+            PrincipalName = principalName;
+            ObjectId = objectId;
+            TenantId = tenantId;
+        }
+
+        /// <summary> The principal type used to represent the type of Active Directory Administrator. </summary>
+        public PostgreSqlFlexibleServerPrincipalType? PrincipalType { get; set; }
+        /// <summary> Active Directory administrator principal name. </summary>
+        public string PrincipalName { get; set; }
+        /// <summary> The objectId of the Active Directory administrator. </summary>
+        public string ObjectId { get; set; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#d4cebe383870410cab2e14d0709f78bb).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#9f6fc0ae7c8e497e98d7979dadf97a4c).

Reply:
`ObjectId` should be a GUID.

Reply:
Fixed.